### PR TITLE
Bump Camel to 1.0.0-M6

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
@ppalaga FYI

**Comment by geoand** [comment]
I have no idea why CI is failing...

File: bom/deployment/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
             <!-- Camel -->
             <dependency>
                 <groupId>org.apache.camel.quarkus</groupId>
-                <artifactId>camel-quarkus-parent</artifactId>
+                <artifactId>camel-quarkus-bom</artifactId>
```
Comment (ppalaga) [inline bom/deployment/pom.xml:40]: This is a deployment BOM. So we should rather import `camel-quarkus-bom-deployment`. There is a fix in #49, BTW.

Comment (geoand) [reply bom/deployment/pom.xml:40]: You would know best, I just used whatever worked :). I updated the PR.

**Comment by geoand** [comment]
I'll rebase this onto the latest `master` since #49 was merged.

**Comment by geoand** [comment]
PR rebased

**Comment by geoand** [comment]
Superseded by #51

**Pull request closed without merging**